content/en/_index.md

```diff
@@ -275,22 +308,66 @@ async function listInfoTypes() {
   });
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+import (
+	"context"
+	"fmt"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// listInfoTypes lists the types of sensitive information the DLP API supports.
+func listInfoTypes(w io.Writer) {
+	ctx := context.Background()
+	// Initialize client that will be used to send requests. This client only
+	// needs to be created once, and can be reused for multiple requests. After
+	// completing all of your requests, call the "Close" method on the client to
+	// safely clean up any remaining background resources.
+	c, err := dlp.NewClient(ctx)
+	if err != nil {
+		// TODO: Handle error.
+	}
+	defer c.Close()
+
+	// Construct the request to be sent by the client
```

Comment:
This line maybe feels helpful for GAPIC/Apiary but is overkill for a manual client IMO

Reply:
This is in the current canonical: https://github.com/GoogleCloudPlatform/golang-samples/blob/68e04de0b13bfad4d34c9556ca66ed40655dcdd4/dlp/snippets/inspect/inspect_string.go#L40